src/event-handler.js

```diff
@@ -23,7 +22,8 @@ EventHandler.prototype.logEvent = function(event) {
         window['optimizely'].push(optimizelyEvent);
     }
 
-    if (optimizelyFullStackEvents.events[event.EventName]) {
+    // if optimizely full stack is being used
+    if (window.optimizelyClientInstance) {
```

Comment:
similarly, in this file, rename to `optimizelyWebXEvents` and `optimizelyWebXEvent` where applicable